### ISSUE TITLE
Clone specific buffs

### ISSUE THREAD

**Issue by elzibubble** [Lailloken/Exile-UI#492]
I'm running Charged Staff in PoE2 and staring at topleft to track my power charges and buff. Trouble is, I get short buffs that last a few seconds sometimes too, so they're not in dependable positions. It should be possible to image search for the buff icon (being careful to only use the portion not overlaid by the stack counter).

**Comment by elzibubble** [comment]
I wonder how many versions there are? Could depend on resolution + a number of factors though. Frustrating, I'm not holding my breath for GGG to make buffs easier to manage. Thank you for the quick response.

**Comment by Lailloken** [comment]
I just remembered something tangentially related. I was playing around with a feature to scan and sum up trade-window contents, and that UI has the same scaling/rendering issue: https://youtu.be/rJT9WElo_Zo?feature=shared&t=14

1440p has 4 different versions for a currency stack, and 1080p had 8 IIRC.

**Comment by elzibubble** [comment]
Yeah I can't imagine running 8 image searches will be too fast, even in a small area

**Comment by Dagothur** [comment]
+1 for this. I'm only interested in one buff and it's constantly switching between buff positions 3-4.

I haven't looked under the hood of what you're doing but if you used AHK's GDI library the function to find an image / image search comes built in with a variance setting. Extending the existing functionality to seed the clone region's xy based on a target image and search region xywh would be awesome.

Or an alternative for my personal use case would be hotkey(s) to enable some sort of toggling or cycling of positions.
